vpr/src/route/rr_graph.cpp

```diff
@@ -785,7 +787,9 @@ void create_rr_graph(const t_graph_type graph_type,
                            directs, num_directs,
                            &det_routing_arch->wire_to_rr_ipin_switch,
                            is_flat,
-                           Warnings);
+                           Warnings,
+                           router_opts
```

Comment:
Similar to my previous comments, this should also only pass `route_verbosity`. Look at the other arguments to this function. Notice how they also just pass in the members of `router_opts`. This is just good practice and what VPR tends to do. Passing these router opts deeper into method can make things more confusing.